lib/core/src/lib/pipes/localized-date.pipe.ts

```diff
@@ -37,14 +36,12 @@ export class LocalizedDatePipe implements PipeTransform, OnDestroy {
 
     private onDestroy$ = new Subject<boolean>();
 
-    constructor(public userPreferenceService?: UserPreferencesService,
-                public appConfig?: AppConfigService) {
-
+    constructor(public userPreferenceService?: UserPreferencesService, public appConfig?: AppConfigService) {
         if (this.userPreferenceService) {
             this.userPreferenceService
                 .select(UserPreferenceValues.Locale)
                 .pipe(takeUntil(this.onDestroy$))
```

Comment:
takeUntil should be replaced with takeUntilDestroyed here

Reply:
@dominikiwanekhyland not addressed

Reply:
Yes, it's left on purpose. LocalizedDatePipe is created as a class object in CardViewDateItemModel, which is than created in many places as a js object, without injector. Changing this to takeUntilDestroy() would require lots of refactoring, which I think is unecessary - in the end takeUntil(this.onDestroy$) is doing it's

Reply:
ok then, thank you for explanation